### PR TITLE
Feat(Query): Volume Mounted In Multiple Containers for docker compose

### PR DESCRIPTION
**Pull request by cxAndreFelicidade**
**Proposed Changes**
- Query: Volume Mounted In Multiple Containers for docker compose

I submit this contribution under the Apache-2.0 license.

### CODE REVIEW

File: assets/queries/dockerCompose/volume_mounted_in_multiple_containers/metadata.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "id": "baa452f0-1f21-4a25-ace5-844e7a5f410d",
+  "queryName": "Volume Mounted In Multiple Containers",
+  "severity": "HIGH",
+  "category": "Build Process",
+  "descriptionText": "Volume is mounted in multiple containers. Volume mounts should not be shared.",
```
Comment (rafaela-soares) [inline assets/queries/dockerCompose/volume_mounted_in_multiple_containers/metadata.json:6]:
```suggestion
  "descriptionText": "Volume mounts should not be shared, which means that 'propagation' should not be set to 'shared', 'rshared', 'slave', or 'rslave'",
```

File: assets/queries/dockerCompose/volume_mounted_in_multiple_containers/query.rego [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    propagation == possibilities[p]
+    
+	result := {
+    	"debug": sprintf("%s",[propagation]),
+		"documentId": sprintf("%s", [resource.id]),
+		"searchKey": sprintf("services.%s.volumes.bind.propagation",[name]),
+		"issueType": "IncorrectValue",
+		"keyExpectedValue": "Volumes are not being mounted in multiple containers",
+		"keyActualValue": sprintf("Volumes are being mounted in multiple containers, mode: %s", [possibilities[p]]),
+		"searchLine": common_lib.build_search_line(["services", name, "volumes", "bind", "propagation"], []),
```
Comment (rafaela-soares) [inline assets/queries/dockerCompose/volume_mounted_in_multiple_containers/query.rego:21]:
```suggestion
        "searchLine": common_lib.build_search_line(["services", name, "volumes", v, "bind", "propagation"], []),
```

File: assets/queries/dockerCompose/volume_mounted_in_multiple_containers/query.rego [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+	resource := input.document[i]
+	service_parameters := resource.services[name]
+    volumes := service_parameters.volumes
+    volume := volumes[v]
+    propagation := volume.bind.propagation
+    possibilities := {"shared", "rshared", "slave", "rslave"}
+    propagation == possibilities[p]
+    
+	result := {
+    	"debug": sprintf("%s",[propagation]),
```
Comment (rafaela-soares) [inline assets/queries/dockerCompose/volume_mounted_in_multiple_containers/query.rego:15]:
```suggestion
```

**Review by rafaela-soares — CHANGES_REQUESTED** [review]

**Review by rafaela-soares — APPROVED** [review]
LGTM 🚀

File: assets/queries/dockerCompose/volume_mounted_in_multiple_containers/test/positive_expected_result.json [hunk 4]
```diff
@@ -0,0 +1,26 @@
+[
+    {
+      "queryName": "Volume Mounted In Multiple Containers",
+      "severity": "HIGH",
+      "line": 15,
+      "filename": "positive1.yaml"
```
Comment (joaorufi) [inline assets/queries/dockerCompose/volume_mounted_in_multiple_containers/test/positive_expected_result.json:6]:
```suggestion
      "fileName": "positive1.yaml"
```

File: assets/queries/dockerCompose/volume_mounted_in_multiple_containers/test/positive_expected_result.json [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+    },
+    {
+      "queryName": "Volume Mounted In Multiple Containers",
+      "severity": "HIGH",
+      "line": 15,
+      "filename": "positive2.yaml"
```
Comment (joaorufi) [inline assets/queries/dockerCompose/volume_mounted_in_multiple_containers/test/positive_expected_result.json:12]:
```suggestion
      "fileName": "positive2.yaml"
```

File: assets/queries/dockerCompose/volume_mounted_in_multiple_containers/test/positive_expected_result.json [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+    },
+    {
+      "queryName": "Volume Mounted In Multiple Containers",
+      "severity": "HIGH",
+      "line": 15,
+      "filename": "positive3.yaml"
```
Comment (joaorufi) [inline assets/queries/dockerCompose/volume_mounted_in_multiple_containers/test/positive_expected_result.json:18]:
```suggestion
      "fileName": "positive3.yaml"
```

File: assets/queries/dockerCompose/volume_mounted_in_multiple_containers/test/positive_expected_result.json [hunk 7]
```diff
@@ -0,0 +1,26 @@
...
+    },
+    {
+      "queryName": "Volume Mounted In Multiple Containers",
+      "severity": "HIGH",
+      "line": 15,
+      "filename": "positive4.yaml"
```
Comment (joaorufi) [inline assets/queries/dockerCompose/volume_mounted_in_multiple_containers/test/positive_expected_result.json:24]:
```suggestion
      "fileName": "positive4.yaml"
```

**Review by joaorufi — CHANGES_REQUESTED** [review]

**Review by joaorufi — APPROVED** [review]

**Pull request merged**